### ISSUE TITLE
MP GO:0000964 mitochondrial RNA 5'-end processing

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#14873]
MP GO:0000964 mitochondrial RNA 5'-end processing

please add parent GO:0000963 mitochondrial RNA processing

**Comment by pgaudet** [comment]
Changes: 

* mRNA processing *NEW logical definition*
  'RNA metabolic process'
   and ('has participant' some mRNA)

* mitochondrial RNA processing
'RNA processing'
 and ('part of' some 'mitochondrial gene expression') REMOVE
 and ('occurs in' some mitochondrion)
 and ('has input' some 'ribonucleic acid')

* mitochondrial tRNA processing
'RNA processing'
 and ('part of' some 'mitochondrial gene expression') REMOVE
 and ('occurs in' some mitochondrion)
 and ('has input' some tRNA)

NOTE: The parent RNA processing is 'part of' gene expression, so that relation was redundant. Thanks, Pascale

**Comment by ukemi** [comment]
Are these really necessary and sufficient, particularly the first one?

**Comment by pgaudet** [comment]
Hi @ukemi 
Thanks for pointing that out - I miscopied the changed (it's really hard to follow these edits in the diff).

In fact I left RNA processing without a definition (I wasn't able to do a definition that was different from RNA metabolic process). 

I hope that makes more sense. 

Thanks, Pascale

**Comment by ValWood** [comment]
I don't think that worked.

mitochondrial RNA processing is still not part of mitochondrial gene expression?

**Comment by pgaudet** [comment]
![image](https://user-images.githubusercontent.com/4782928/35214903-022acfbc-ff63-11e7-8783-4a09e5dd52b1.png)

Isn't this what you wanted ?

**Comment by ValWood** [comment]
This is correct but still has a parent missing.

"RNA 5' end processing" is a part_of "gene expression"

Therefore
"mitochondrial RNA 5' end processing"

should be part of "mitochondrial gene expression"

(we have "mitochondrial gene expression" in our slim because its a largely independent biological "module"  and I spotted it because  "mitochondrial RNA 5' end processing"
annotations are not slimming.

**Comment by ValWood** [comment]
Actually, the issue is that "mitochondrial RNA processing" does not have the paretn "RNA processing"

It seems odd that the reasoner does not find that one?

![rna proc](https://user-images.githubusercontent.com/7359272/35217694-16542064-ff64-11e7-87d9-7f66793c88cf.jpg)

**Comment by ValWood** [comment]
Well fititng the definition of "RNA processing"

Any process involved in the conversion of one or more primary RNA transcripts into one or more mature RNA molecules.

it should be, because processing is part of gene expression.

**Comment by ValWood** [comment]
seems so....
Any process involved in forming the mature 3' end of an RNA molecule.

**Comment by ValWood** [comment]
Anything  pre-translation is gene expression:


gene expression
Biological Process

Definition (GO:0010467 GONUTS page)
The process in which a gene's sequence is converted into a mature gene product or products (proteins or RNA). This includes the production of an RNA transcript as well as any processing to produce a mature RNA product or an mRNA or circRNA (for protein-coding genes) and the translation of that mRNA or circRNA into protein.

**Comment by pgaudet** [comment]
I now also added 'part of' some 'gene expression' to RNA 3'-end processing. 

However I noticed other oddities in that branch: 
1) 'part of' some 'gene expression' seems to be missing from the following terms: 

'RNA capping'
'RNA splicing'
'rRNA processing'
'snoRNA processing'
'snRNA processing'
'tRNA processing'
'tRNA-type intron splice site recognition and cleavage'

2) 'RNA capping' should be a child of 'RNA 5'-end processing'

3) 'RNA polyadenylation' should be a child of 'RNA 3'-end processing'

**Comment by ValWood** [comment]
I  think so...

Also I'm still not seeing 
"mitochondrial RNA processing" is part-of  "mitochondrial gene expression" in QuickGO or PomBase, but that may just be a time -lag?

https://www.ebi.ac.uk/QuickGO/term/GO:0000963

**Comment by ValWood** [comment]
![mitochondrial gene expression](https://user-images.githubusercontent.com/7359272/35565897-49fce2bc-05b7-11e8-9e71-0b35cbcd6452.jpg)
